spec/controllers/game_controller_spec.rb

```diff
@@ -1,18 +1,18 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe GameController, type: :controller do
-  describe '#new' do
+  describe "#new" do
     context do
       before { get :new }
       it { expect(response.status).to eq 200 }
     end
   end
 
-  describe '#init' do
+  describe "#init" do
     let(:params) do
       ActionController::Parameters.new({
-                                           x: 'Player x',
-                                           o: 'Player O'
+                                           x: "Player x",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.